### PR TITLE
docs: introduction

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Intro docs.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+The Kaltura Player exposes two different players: the *Kaltura OVP Player* and *Kaltura Cloud TV Player*. Each player integrates its related packages, as you can see in the following table:
+
+
+|  | PlayKit JS | PlayKit JS Providers|PlayKit JS UI| PlayKit JS DASH | PlayKit JS HLS| PlayKit JS Youbora |  PlayKit JS KAnalytics|  PlayKit JS OTT Analytics|
+| --- |---|---|---|---|---|---| ---| --- |
+| OVP Player | V | OVP | V | V | V | V | V |  |
+|  Cloud TV Player | V |  OTT | V | V | V | V |  | V | 
+
+Next, let's look at how to [get started]() by creating a player using the Player API set.
```
Comment (AnnaAleksandrowicz) [inline docs/introduction.md]: Dan - what you think about linking from here to your quick start guide (or to both yours and Yair's?)?

File: docs/introduction.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+The Kaltura Player integrates different packages into one single package, creating a powerful media player.
+
+The Kaltura Player integrates:
+* [PlayKit JS ](https://github.com/kaltura/playkit-js) as the core library.
```
Comment (OrenMe) [inline docs/introduction.md]: substitute `as` with just `-`, e.g. `PlayKit JS - the core library`

File: docs/introduction.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+The Kaltura Player integrates different packages into one single package, creating a powerful media player.
```
Comment (OrenMe) [inline docs/introduction.md]: The Kaltura Player takes on a highly modular approach for creating a powerful media player.
Each functionality of the player is isolated to separate package focused on delivering specific set of abilities.
This approach enables high Modularity, Extensibility, Simplicity and Maintainability.

@AnnaAleksandrowicz - WDYT?

Comment (AnnaAleksandrowicz) [reply docs/introduction.md]: The Kaltura Player utilizes a highly modular approach for creating a powerful media player.
Each functionality of the player is isolated into separate packages, which are designed to deliver a specific set of abilities.
This approach enables extensibility, simplicity and easy maintenance..
@OrenMe - see if you are OK with my suggested changes - also, we already talk about modularity in the first part, so no point repeating it.

Comment (OrenMe) [reply docs/introduction.md]: so @dan-ziv, please use @AnnaAleksandrowicz suggestion

**Pull request closed without merging**